Drukhari/Readme.md

```diff
@@ -0,0 +1,96 @@
+# Drukhari Index v1.0
+## General Information
+* This Index contains all units from the current `Index: Drukhari`
+  
+* **`Leaders`** are all separate unit profiles acting as if they were a sole unit on the battlefield, they include their **`Leader`** ability but this should be left inactive unless simulating that effect.
+  
+* All unit profiles that have a squad size have been created at their minimum size and the following datasheets have alternate sizes as a separate profile:
+    - Talos (x1, x2)
+    - Kabalite Warriors (x10, x5 special weapons, x5 splinter rifles)
+  
+* Some bodyguard + leader combinations are included:
+    - Court of the Archon + Archon
+    - Incubi x5 + Drazhar
+    - Kabalite Warriors x10 + Archon
+    - Wracks x5 + Haemonculus
+    - Wyches x10 + Lelith Hesperax
+    - Wyches x10 + Succubus
+  
+* All units that have the `Damaged: 1-X Wounds Remaining` ability have a `-1 to hit roll (if attacker)` ability in their unit profile that can be enabled.
```

Comment:
* Remove this line or add the `-1 to hit roll (if attacker)` profile ability to the following Units: `Ravager`, `Voidraven Bomber`.